src/controllers/confirm.sic.code.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Request, Response } from "express";
+import { TASK_LIST_PATH, urlParams } from "../types/page.urls";
+import { Templates } from "../types/template.paths";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const companyNumber = req.params[urlParams.PARAM_COMPANY_NUMBER];
+    const backLinkUrl = TASK_LIST_PATH.replace(`:${urlParams.PARAM_COMPANY_NUMBER}`, companyNumber);
```

Comment:
Could consider using the [recently added function to populate the URL](https://github.com/companieshouse/confirmation-statement-web/pull/152/files). Could be something like
` const backLinkUrl = getUrlWithCompanyNumber(TASK_LIST_PATH, companyNumber);`